librz/core/cmd_type.c

```diff
@@ -1735,3 +1822,162 @@ RZ_IPI int rz_cmd_type(void *data, const char *input) {
 	}
 	return true;
 }
+
+RZ_IPI RzCmdStatus rz_type_cc_list_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	const char *cc = argc > 1 ? argv[1] : NULL;
+	if (cc) {
+		types_cc_print(core, cc);
```

Comment:
considering this can be called also with `tccj <cc>` i might expect it to return JSON as well.